<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>440</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Unit Converter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QListView" name="unitCategoryListView"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="unitConversionArea">
      <item>
       <layout class="QHBoxLayout" name="unitPickerArea">
        <item>
         <layout class="QVBoxLayout" name="fromUnitArea">
          <item>
           <widget class="QListView" name="fromUnitPicker"/>
          </item>
          <item>
           <widget class="QLineEdit" name="fromUnitInput"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="toUnitArea">
          <item>
           <widget class="QListView" name="toUnitPicker"/>
          </item>
          <item>
           <widget class="QLineEdit" name="toUnitOutput">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="resetButton">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionReset"/>
    <addaction name="separator"/>
    <addaction name="actionCopy_Input_to_Clipboard"/>
    <addaction name="actionCopy_Result_to_Clipboard"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionDisplay_All_Digits"/>
    <addaction name="separator"/>
    <addaction name="actionRound_off_to_whole_number"/>
    <addaction name="actionRound_off_to_1_d_p"/>
    <addaction name="actionRound_off_to_2_d_p"/>
    <addaction name="actionRound_off_to_3_d_p"/>
    <addaction name="actionRound_off_to_4_d_p"/>
    <addaction name="actionRound_off_to_5_d_p"/>
    <addaction name="actionRound_off_to_6_d_p"/>
    <addaction name="actionRound_off_to_7_d_p"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDisplay_All_Digits">
   <property name="text">
    <string>Display All Digits</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionCopy_Input_to_Clipboard">
   <property name="text">
    <string>Copy Input to Clipboard</string>
   </property>
  </action>
  <action name="actionCopy_Result_to_Clipboard">
   <property name="text">
    <string>Copy Result to Clipboard</string>
   </property>
  </action>
  <action name="actionRound_off_to_whole_number">
   <property name="text">
    <string>Round off to whole number</string>
   </property>
  </action>
  <action name="actionRound_off_to_1_d_p">
   <property name="text">
    <string>Round off to 1 d.p.</string>
   </property>
  </action>
  <action name="actionRound_off_to_2_d_p">
   <property name="text">
    <string>Round off to 2 d.p.</string>
   </property>
  </action>
  <action name="actionRound_off_to_3_d_p">
   <property name="text">
    <string>Round off to 3 d.p.</string>
   </property>
  </action>
  <action name="actionRound_off_to_4_d_p">
   <property name="text">
    <string>Round off to 4 d.p.</string>
   </property>
  </action>
  <action name="actionRound_off_to_5_d_p">
   <property name="text">
    <string>Round off to 5 d.p.</string>
   </property>
  </action>
  <action name="actionRound_off_to_6_d_p">
   <property name="text">
    <string>Round off to 6 d.p.</string>
   </property>
  </action>
  <action name="actionRound_off_to_7_d_p">
   <property name="text">
    <string>Round off to 7 d.p.</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
